cluster/search/client.py

```diff
@@ -39,7 +43,26 @@ def upload(self, documents):
             doc["_index"] = self.index
         return bulk(self.es, documents)
 
-    def search(self, query, limit=100, only_documents=True):
+    def update(self, query, scroll, updater):
```

Comment:
This is for index-wide changes. It'll retrieve each document and reindex it. So no, there's no "update" support.